src/main/java/com/google/api/codegen/transformer/ModelTypeTable.java

```diff
@@ -173,4 +173,8 @@ public String getImplZeroValueAndSaveNicknameFor(TypeRef type) {
   public Map<String, TypeAlias> getImports() {
     return typeTable.getImports();
   }
+
+  public TypeTable getTypeTable() {
```

Comment:
Yeah, it's needed for it to work. Essentially, `ModelTypeTable` is a decorator around `TypeTable`.